handlers/auth_code_request_update.go

```diff
@@ -154,3 +159,16 @@ func UpdateAuthCodeRequest(authCodeSvc *service.AuthCodeService, authCodeReqSvc
 
 	})
 }
+
+func sendConfirmationEmail(emailAddress string, r *http.Request, w http.ResponseWriter) {
+	// Send confirmation email
+	if err := handleEmailKafkaMessage(emailAddress, r); err != nil {
+		log.ErrorR(r, err, log.Data{"email address": emailAddress})
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	log.Info("confirmation email sent to customer", log.Data{
```

Comment:
Use InfoR to include request in log
```suggestion
	log.InfoR(r, "confirmation email sent to customer", log.Data{
```